backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
         )
         self._general_conf = config["general"]
 
-    def configuration_hook(
+    def configuration_hook(  # type: ignore
```

Comment:
I suggest to be specific with what rule to ignore:

```suggestion
    def configuration_hook(  # type: ignore[override]
```